appendix.zh.md

```diff
@@ -0,0 +1,39 @@
+RMS亏待女性，使女性感到不适、不安全、不受欢迎由来已久。关于RMS在MIT的所作所为，请见 [0]。
+
+[0]: https://selamjie.medium.com/remove-richard-stallman-appendix-a-a7e41e784f88
+
+RMS关于强奸和儿童性法律的观点在2019年秋季广受讨论，见 Selam G 的文章。[1]特别令人骨寒的是，当RMS谈到Marvin Minsky对Jeffrey Epstein的一个贩卖受害者（Virginia Giuffre）进行性侵犯的指控时，他说：「但最可信的情况是，她向他展示自己是完全自愿的。」[2]RMS认为这并非性侵犯，因为「『强奸』需要有暴力」，而新闻报告却「没有提到这一点，只提到两者进行了性交。」[3]与其进一步讨论这个问题，不如关注他在个人网站上关于未成年人「完全自愿」的看法。[4]（注：一些当时的新闻报道错误表述了RMS对Minsky的看法，但这个错误报告和他在个人网站上一贯表示的立场一致。）
+
+[1]: https://web.archive.org/web/20210325013429/https://selamjie.medium.com/remove-richard-stallman-fec6ec210794
+[2]: https://web.archive.org/web/20210325013629/https://www.vice.com/en/article/9ke3ke/famed-computer-scientist-richard-stallman-described-epstein-victims-as-entirely-willing
+[3]: https://web.archive.org/web/20210325013629/https://www.vice.com/en/article/9ke3ke/famed-computer-scientist-richard-stallman-described-epstein-victims-as-entirely-willing
+[4]: https://web.archive.org/web/20210325013706/https://stallman.org/archives/2018-jul-oct.html#23_September_2018_(Cody_Wilson)
+
+RMS经常对「忽视青少年完全自愿，将与青少年性交定为『强奸』的虚伪法律」发表评论。[5]他认为美国法律和苏丹法律在强奸方面没有区别，认为「婚内强奸」和「年龄不足即强奸」的概念都是「歪曲了『强奸』的意义」。[6]
```

Comment:
@ehashman I should mention that this Sudan part is a pretty important difference from the other translation mainly because the original is not written well enough. The statement

> He compares United States law to Sudanese law in saying that “US laws that define 'rape' to include voluntary sex with under N years of age (where N varies)” and that “both laws falsify the meaning of ‘rape.’”

does not highlight the also important issue that is a disregard of marital rape.